src/main/java/com/tmh/controller/admin/UserController.java

```diff
@@ -76,6 +83,8 @@ public String submitAddOrUpdateUser(@ModelAttribute("userForm") User user, Bindi
 		userService.saveOrUpdate(user);
 		model.addAttribute("user", user);
 		
+		model.addAttribute("msg", "User is added / updated!");
```

Comment:
` "User is added / updated!"` Dùng i18n đi chú

Reply:
Chị ơi cái này bắt buộc phải dùng i18n ạ? :))

Reply:
ukm, label ở html chị tha, vì nhìu quá, nhát, loạn, mệt...
còn mấy message này thì cần
sau này tụi em làm web, thì I18n mọi text nhé, text nào hiển thị ở view đều phải làm I18n nhé

Reply:
Vâng oki chị ơi !! :))